testing/scripts/capture_derive_snapshots.py

```
@@ -0,0 +1,131 @@
+        await _wait_app_ready(page)
+        await page.wait_for_selector(".tool-panel[data-tool='derive'].active", timeout=3000)
+        await _scroll_panel(page, "derive")
+    else:
+        raise ValueError(state)
+
+    await capture_snapshot(page, name)
+    print(f"  captured: {name}")
+
+
+async def main() -> int:
+    server = subprocess.Popen(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)